base/exports.jl

```diff
@@ -994,7 +994,6 @@ export
 # loading source files
     __precompile__,
     evalfile,
-    include,
```

Comment:
could have mentioned that in the commit message or PR description, I suspect that's totally unclear to anyone other than you